packages/manager/modules/pci/src/projects/project/translations/Messages_fr_FR.json

```
@@ -19,5 +19,6 @@
   "pci_projects_project_welcome": "Bienvenue sur la nouvelle interface Public Cloud",
   "pci_projects_project_keystone_upgrade": "Changement important le 23 juin 2020, merci d'anticiper dès maintenant le passage à <strong>Keystone v3</strong>.",
   "pci_projects_project_keystone_upgrade_more": "En savoir plus",
-  "pci_projects_project_quota_warning_message": "Par mesure de sécurité, votre projet est soumis à un quota. Si vous voulez augmenter ce quota, rendez-vous sur la page <a data-ui-sref=\"{{ quotaUrl }}\">Quota and localization</a>."
+  "pci_projects_project_quota_warning_message": "Par mesure de sécurité, votre projet est soumis à un quota. Si vous voulez augmenter ce quota, rendez-vous sur la page <a data-ui-sref=\"{{ quotaUrl }}\">Quota and localization</a>.",
+  "pci_projects_project_quota_threshold_warning_message": "Vos niveaux de quotas arrivent à leur maximum, <a href=\"{{ quotaUrl }}\">cliquez ici pour gérer vos quota</a>."
```

I think we can have something like this:
```suggestion
  "pci_projects_project_quota_threshold_warning_message": "Vos niveaux de quotas arrivent à leur maximum.",
  "pci_projects_project_quota_threshold_warning_message_link": "Cliquez ici pour gérer vos quota."
```

---

Thanks for the comment. It's very valid and useful. QC is done and I have already initiated the translations. It's almost done. I can not make this change in this PR. However, I will handle this in a separate PR when this PR is merged.

---

@radireddy I don't see any issue here applying this specific suggestion, it's not major content changes and quickly adds a patch for customer using French language in the manager. We might even not need to resubmit translation for this one as it might be corrected during translation process